packages/@core.keystone/test.utils.js

```diff
@@ -97,151 +114,194 @@ const prepareNextExpressApp = async (dir) => {
     return { app }
 }
 
-const makeFakeClient = async (app, server) => {
-    const request = require('supertest')
-    const port = server.address().port
-    const protocol = app instanceof https.Server ? 'https' : 'http'
-    const serverUrl = protocol + '://127.0.0.1:' + port
+/**
+ * @param {Error} e
+ * @param {Object[]} e.graphQLErrors
+ * @param {ServerError} e.networkError
+ * @param {String} e.message
+ * @param {*} e.extraInfo
+ * @param {String} e.stack
+ * @returns {{errors: []}}
+ */
+function gqlCatchedErrorsHandler (e) {
+    const errors = []
+
+    if (e.graphQLErrors && e.graphQLErrors.length > 0) {
+        e.graphQLErrors.map((graphQLError) => {
+            errors.push(graphQLError)
+        })
+    }
 
-    const client = request(server)
+    if (e.networkError) {
+        if (e.networkError.result && e.networkError.result.errors) {
+            e.networkError.result.errors.forEach((err) => errors.push(err))
+        }
+    }
 
-    let cookies = {}
+    return { errors }
+}
 
+/**
+ * @param {function} callable
+ * @param {Object} params
+ * @returns {Promise<Object>}
+ */
+async function doGqlRequest (callable, params) {
+    try {
+        return await callable({
+            ...params,
+            // About error policies see https://www.apollographql.com/docs/react/v2/data/error-handling/#error-policies
+            errorPolicy: 'all',
+            fetchPolicy: 'no-cache',
```

Comment:
are you sure that you need to disable cache here too?

Reply:
Yes, because we have some tests where the query returns the same result even after values have changed in database.
Example: `apps/condo/domains/user/schema/ConfirmPhoneAction.test.js:193` test `should change sms code when resend is invoked`